test/CC/ShellCheck/FingerprintSpec.hs

```diff
@@ -71,7 +71,11 @@ fingerprintSpecs = describe "issueFingerprint" $ do
         fp1 `shouldBe` fp2
 
 fingerprint :: Integer -> Code -> Text -> Text
-fingerprint ln code = issueFingerprint $ PositionedComment (position ln) (comment code)
+fingerprint ln code =
+  issueFingerprint $ PositionedComment (position ln) unused (comment code)
+  where
+    unused :: a
+    unused = error "end position comment not implemented"
```

Comment:
This is never used by our code and is only related to tests, so I've opted to error out if we ever end up touching it. 

It'll go away when we implement https://github.com/filib/codeclimate-shellcheck/issues/61